src/components/feedback/ToastMessages.tsx

```diff
@@ -52,9 +52,9 @@ type ToastMessageItemProps = {
  */
 function ToastMessageItem({ message, onDismiss }: ToastMessageItemProps) {
   // Capitalize the message type for prepending; Don't prepend a message
-  // type for "notice" messages
+  // type for "notice" or hidden messages
```

Comment:
This comment explains what the code does, but not why. The prefix is omitted for notices because that's the default, and for hidden messages because the type is assumed to be obvious from the context.

The main problem with the prefix is just that it takes up time (when read) or space (on screen) that feels unnecessary. This is especially true because all the callouts have an icon to indicate their type.